src/components/CommentTeaser/CommentTeaser.js

```diff
@@ -198,21 +201,14 @@ export const CommentTeaser = ({
                     discussion={discussion}
                     passHref
                   >
-                    <a {...linkRule}>
-                      {inQuotes(discussion.title)}
-                      {newPage && (
-                        <span {...styles.icon}>
-                          <NewPage size={ICON_SIZE} fill={colors.disabled} />
-                        </span>
-                      )}
-                    </a>
+                    <a {...linkRule}>{inQuotes(discussion.title)}</a>
```

Comment:
`linkRule` should also be deprecated with `colors`. You can use `A` from `Typography`: `import { A } from '../Typography/'`

```suggestion
                    <A>{inQuotes(discussion.title)}</A>
```

`linkRule` was necessary way back before Next.js had an `passHref` on `Link` and mandated the usage of native `a` tags – not allowing custom (styled) components. There is another occurrence in `src/components/Social/Tweet.js` which needs adaption too.